ApplicationLogs/ApplicationLogs.csproj

```diff
@@ -14,7 +14,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Neo" Version="3.0.0-CI00804" />
+    <ProjectReference Include="..\LevelDBStore\LevelDBStore.csproj" />
```

Comment:
Should be great if the plugin use the current Store. Something like a `Clone(string path)` in `IStore`

Reply:
What do you mean?

Reply:
I mean that if the node is using RocksDB, this plugin use mandatory LevelDB

Reply:
This plugin uses leveldb with a different store. We can improve it in the future.